Add legacy Trac landing page and redirects (#823)
piwik/developer-documentation: docs/trac-ticket-archive.md

---
category: TracTicketArchiveCategory
---

> #Legacy Ticket Archive

Reviewer: ```suggestion
# Legacy ticket archive

```

> ##Trac – Legacy Issue Tracker

Reviewer: ```suggestion
## Trac – Legacy issue tracker

```

The ticket you tried to access (e.g., /trac/ticket/1647) is no longer available.
We’ve migrated away from Trac and now use GitHub for improved collaboration and transparency.

The old Trac system has been permanently shut down, and direct access to individual Trac tickets is no longer possible.

> ##[GitHub – Current Issue Tracker](https://github.com/matomo-org/matomo/issues)

Reviewer: ```suggestion
## GitHub – Current issue tracker

```

> You can [create a bug report](https://github.com/matomo-org/matomo/issues) or check the status of an existing issue in our current tracking system.

Reviewer: ```suggestion
We now use [GitHub](https://github.com/matomo-org/matomo/issues) to track issues. You can [create a new issue](https://github.com/matomo-org/matomo/issues/new/choose) or check the status of an existing issue in our current tracking system.
```
